Tests/CLITests/Subcommands/Build/CLIBuilderLocalOutputTest.swift

```diff
@@ -0,0 +1,287 @@
+//===----------------------------------------------------------------------===//
+// Copyright © 2025 Apple Inc. and the container project authors. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//===----------------------------------------------------------------------===//
+
+//
+
+import Foundation
+import Testing
+
+extension TestCLIBuildBase {
+    class CLIBuilderLocalOutputTest: TestCLIBuildBase {
+        override init() throws {
+            try super.init()
+        }
+
+        deinit {
+            try? builderDelete(force: true)
+        }
+
+        @Test func testBuildLocalOutputBasic() throws {
+            let tempDir: URL = try createTempDir()
+            let dockerfile: String =
+                """
+                FROM scratch
+
+                ADD testfile.txt /hello.txt
+                """
+            let context: [FileSystemEntry] = [
+                .file("testfile.txt", content: .data("Hello from local build\n".data(using: .utf8)!))
+            ]
+            try createContext(tempDir: tempDir, dockerfile: dockerfile, context: context)
+
+            let outputDir = tempDir.appendingPathComponent("local-output")
+            let imageName = "local-build-test:\(UUID().uuidString)"
+
+            let response = try buildWithLocalOutput(tag: imageName, tempDir: tempDir, outputDir: outputDir)
+
+            // Verify the build succeeded
+            #expect(response.contains("Successfully exported to"), "Expected successful local export message")
+
+            // Verify the output directory was created
+            #expect(FileManager.default.fileExists(atPath: outputDir.path), "Expected local output directory to exist")
+
+            // Verify the output contains expected structure
+            let contents = try FileManager.default.contentsOfDirectory(atPath: outputDir.path)
+            #expect(!contents.isEmpty, "Expected local output directory to contain files")
+        }
+
+        @Test func testBuildLocalOutputWithContext() throws {
```

Comment:
What is this test testing differently from the test above?

Reply:
This just checks that the local output dir exists. The other one looks for actual files inside the local output dir

Reply:
The test above captures that too though right?